lib/health_checking/health_checks/issue_commentable.rb

```diff
@@ -0,0 +1,48 @@
+module HealthChecks
+  class IssueCommentableCheck
+    include HealthCheck
+
+    name 'issue_comment_check'
+    weight :info
+    authorization :all
+    message 'Login if you want to comment on this boards issues'
+
+    ## deps
+    # {
+    #   repo: repo object,
+    #   authorization: :all, :collaborator or :admin,
+    #   logged_in: bool
+    # }
+    ##
+    
+    def perform(deps)
+      return not_authorized unless authorized?(deps)
+      return pass if deps[:logged_in]
+      return failure
```

Comment:
Ok yeah - I was thinking along the same lines, but you took it further (but I like)

My thoughts are another mixin (HealthCheck::Instruments?) that will facilitate the meta nature of perform/authorize behind the scenes